.travis.yml

```diff
@@ -134,6 +135,9 @@ script:
   - typo3cms extension:activate core && typo3cms database:updateschema | grep 'No schema updates were performed'
   - typo3cms extension:setup core && typo3cms database:updateschema | grep 'No schema updates were performed'
   - typo3cms extension:setupactive && typo3cms database:updateschema | grep 'No schema updates were performed'
+  - echo "DROP DATABASE travis_test;" | typo3cms database:import
```

Comment:
true, but in this case I just want to remove/ empty the db to test that setupactive also works with empty db